packages/snaps-execution-environments/jest.config.js

```diff
@@ -2,16 +2,11 @@ const deepmerge = require('deepmerge');
 
 const baseConfig = require('../../jest.config.base');
 
+delete baseConfig.coverageThreshold;
```

Comment:
I'm getting ESLint error if try the suggested code:
```
ESLint: Parsing error: Unexpected token ...
```
Our ESLint config does not support that maybe?

Reply:
I did some research about using the error and using the suggested code. So, what I found is that I would need to change the ESLint config to have `ecmaVersion: 2018`. But, in our root directory there is a `tsconfig.packages.json` which has `"target": "ES2017"`, so that is a mismatch with the required ecma version for ESLint. I think I should leave it without changing these configurations for now.